bundle/manifests/fence-agents-remediation.medik8s.io_fenceagentsremediations.yaml

```diff
@@ -73,5 +73,5 @@ status:
   acceptedNames:
     kind: ""
     plural: ""
-  conditions: []
-  storedVersions: []
+  conditions: null
```

Comment:
interesting... in /config this status is removed completely from the CRDs, here just the list values change :thinking: :shrug: